test/bad_dtls_test.c

```diff
@@ -182,7 +185,7 @@ static int validate_client_hello(BIO *wbio)
     /* Update handshake MAC for second ClientHello (with cookie) */
     if (cookie_found && !EVP_DigestUpdate(handshake_md, data + MAC_OFFSET,
                                           len - MAC_OFFSET))
-            printf("EVP_DigestUpdate() failed\n");
```

Comment:
These calls should never fail really. The callers of `validate_client_hello` do wrap the return value in a `TEST` macro. If we add a TEST_info() for this then we should probably do all the "return 0" statements in this function (and probably the other similar functions). I thought that was a fairly low return on investment - but would do it if you insist.